.github/workflows/docker-lint.yaml

```diff
@@ -0,0 +1,23 @@
+name: Dockerfile Linting
+
+# Run the test suite on pushes (incl. merges) to master and dev
+# Run the test suite when a PR is opened, pushed to, or reopened
+on:
+  push:
+    branches:
+      - docker-cleanup-part-1
```

Comment:
I assume we want to update this before we merge

Reply:
Yeah, we'll uncomment the `pull_request` section here, and also have the job run on changes to the dev and master branches. Goal is to have the linter run on pull requests, and then again post-merge.